documentation/src/main/asciidoc/public/reference/_mapping-projection.adoc

```diff
@@ -142,6 +142,7 @@ Constructor parameters can be annotated with explicit projection annotations suc
 For projections that would normally be <<mapping-projection-inner-inference,inferred automatically>>,
 this allows further customization, for example in a <<search-dsl-projection-field-mapping,field projection>>
 to set the target field path explicitly or to disable value conversion.
+This also allows <<search-dsl-projection-object-mapping-filters,breaking cycles of nested object projections>>.
```

Comment:
maybe something like:
```suggestion
Or for <<search-dsl-projection-object-mapping,object projections>> this also allows <<search-dsl-projection-object-mapping-filters,breaking cycles of nested object projections>>.
```
since we don't mention object projection it feels like you can break a cycle with a field projection 😃